subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -576,6 +578,7 @@ protected void appendContents(TreeFormatter formatter) {
 
     @Override
     public boolean contains(File file) {
+        maybeWarnOnDeprecatedUsage("contains", ProperMethodUsage.RESOLVABLE);
```

Comment:
```suggestion
        maybeWarnOnDeprecatedUsage("contains(File)", ProperMethodUsage.RESOLVABLE);
```

We should probably include the parameter types to avoid conflicts with overloaded methods.

Reply:
Good idea.